### DISCUSSION TITLE
Discussions

### DISCUSSION THREAD

**Discussion by Moffran** [Moffran/calibrated_explanations#34] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this package.

**Comment by tuvelofstrom** [comment]
Let me start by introducing myself! My name is Tuwe Löfström. Together with Helena and several coworkers at Jönköping University, we have developed the Calibrated Explanations package. I am currently working part-time in industry as a Senior Software Engineer with focus on AI adoption and part-time in the University where I mostly do research but also some teaching in Data Analytics, Machine Learning (ML), and related topics. I have been researching and teaching on topics related to ML for many years, focusing on how to improve algorithms to help make better decisions. In later years, my two main focus areas are Uncertainty Quantification as well as Explainable and Interpretable AI. With Calibrated Explanations, both these areas are happily married. 

I am passionate about coding but less proficient in python than I would have liked to be, meaning that there probably are ample opportunities to improve the implementation of Calibrated Explanations that I have not even thought of yet. We warmly welcome contributions from a wider community, including both improvements or adding new functionality. 

We have several ideas for further developments and improvements, some of which are likely to be suitable for research publications. If you are interested in research collaboration around an idea or want to discuss collaborations, do not hesitate to reach out to us.
